airflow/serialization/pydantic/taskinstance.py

```diff
@@ -83,6 +84,7 @@ def validated_operator(x: dict[str, Any] | Operator, _info: ValidationInfo) -> A
 class TaskInstancePydantic(BaseModelPydantic, LoggingMixin):
     """Serializable representation of the TaskInstance ORM SqlAlchemyModel used by internal API."""
 
+    id: str
```

Comment:
did this leak from the future?

Reply:
oh no i see we already have it in TI

Reply:
was added in https://github.com/apache/airflow/pull/43243